docs/cookie_authentication.rst

```diff
@@ -0,0 +1,13 @@
+Cookie authentication
```

Comment:
I am really glad that you are working on this! Thanks :). I did some testing and it seems that when token is only in cookie request.user is an AnonymousUser. Cookie can be seen on request.cookies though. I am not very familiar with the code base so I am not sure whether the middleware or just get_token need extension (or none :) ). I am happy to help with this one.

Reply:
Thanks @mtszsobczak !,
I forgot to read the token from the cookie 😅, I'll update the PR soon.

Reply:
Np :) happy to help out